electron/main/Store/storeConfig.ts

```diff
@@ -17,6 +17,11 @@ export interface LocalLLMConfig extends BaseLLMConfig {
 
 export type LLMModelConfig = OpenAILLMConfig | LocalLLMConfig;
 
+export type LLMGenerationParameters = {
+  maxTokens?: number;
+  temperature?: number;
+};
+
```

Comment:
Can these be always defined? But with default values. Just easier to make parameters required so that we can make assumptions about data later on in the app, and refactoring (when we inevitably get to it) becomes easier as well.